src/data/contexts/DayViewContext.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    useState,
+    Dispatch,
+    SetStateAction,
+    createContext,
+    ReactNode,
+} from 'react';
+import { getArrayOfWeekDatesFromDate } from '../../utils/time';
+
+export interface IDayViewContext {
+    selectedDay: Date;
+    setSelectedDay: Dispatch<SetStateAction<Date>>;
+    displayedWeek: Date[];
+    setDisplayedWeek: Dispatch<SetStateAction<Date[]>>;
+}
+
+interface ContextProviderProps {
+    children: ReactNode;
+}
+
+export const today = new Date();
+
```

Comment:
Not sure if it'd work as expected when importing. Does it work consistently for you?

Reply:
It'll probably be best to check for current date between renders if a new day starts. The way it is now you'll have to refresh the page when the day changes